tests/test_example_dags.py

```diff
@@ -30,6 +30,7 @@ def session():
         "example_amazon_s3_snowflake_transform",
         "example_postgres_render",
         "example_snowflake_partial_table",
+        "astroflow_homes_sql_dag",
```

Comment:
It would be interesting to follow the new naming convention  - so that the `nox` runs this example using a subset of the optional dependencies!